app/services/classification/class_param.rb

```diff
@@ -1,41 +0,0 @@
-module Classification
```

Comment:
Thanks, can you add some of that info on top of `values_hash`, `inherited_values`, `possible_value_orders` and `value_of_key`? I think it makes sense now, it's just not that obvious at first glance what they do so if the next reader has an easier time figuring it out, all the better :smile: